web/controllers/user_controller.ex

```diff
@@ -24,7 +24,9 @@ defmodule Org.UserController do
   end
 
   def show(conn, %{"id" => id}) do
-    user = Repo.get!(User, id)
+    user = User
+            |> Repo.get!(id)
```

Comment:
I view the pipe operator as a way of already indenting the next action, and another indentation on top of that makes the next action seem far away haha